e2-studio-star-rx72n-firmware/libs/rx_hcsr04/src/rx_hcsr04_icu.c

```
@@ -0,0 +1,127 @@
+  const uint8_t vector = k_vector_base + irq_num;
+
+  /* Disable interrupt in IER register */
+  const uint8_t ier_index = vector / k_bits_per_ier;
+  const uint8_t ier_bit   = vector % k_bits_per_ier;
+  icu()->ier[ier_index] &= ~(1 << ier_bit);
+
+  /* Clear any pending interrupt flag */
+  icu()->ir[vector] = 0;
+
+  /* Disable digital filter */
+  const rx_err_t err = rx_irq_filter_disable(irq_num);
```

Clang (14.0.6)

[warning] variable 'err' is not initialized

(cppcoreguidelines-init-variables)